html/css/styles.css

```diff
@@ -838,6 +838,13 @@ a.interface-upup:hover, a.interface-updown:hover, a.interface-admindown:hover {
   margin: 8px;
 }
 
+.infobox pre {
```

Comment:
will make it a new class - i did it this way so the bg colour is the same as infobox so if im specifying the colour, i feel it should be with the rest of the CSS instead of hidden away in code somewhere

new change coming shortly